addons/web/static/src/js/core/ajax.js

```diff
@@ -26,13 +26,6 @@ function _genericJsonRpc (fct_name, params, settings, fct) {
     var result = xhr.then(function(result) {
         core.bus.trigger('rpc:result', data, result);
         if (result.error !== undefined) {
-            if (result.error.data.arguments[0] !== "bus.Bus not available in test mode") {
-                if (result.error.data.exception_type === "user_error") {
-                    console.log("Server application error", JSON.stringify(result.error));
```

Comment:
I am not completely sure but if the notification system is trusted there is no need to log the errors anymore.
But I think that it is usefull for runbot tests, the console.error will mark a test as failed.
If i leave it there i would console.error the internal_error and leave the rest as console.log.
Meaning I would have to drop my internal_error rpc test since it will trigger a console.error.
Or find a way to ignore the console.error during my tests.

What do you think ?

Reply:
After further investigations I can conclude that these lines are useless since the Logger in ir_http will make runbot red for the 500 and 403 RPC errors.